### DISCUSSION TITLE
Unavailability of redistribution packages for MOOSE installation in Centos

### DISCUSSION THREAD

**Discussion by sansugath** [idaholab/moose#22819] [Q&A Getting Started] [answered]
Dear all

I am attempting to install MOOSE in Centos. In the instructions for installation in Centos, downloading redistributable packages for  CentOS is mentioned. The URL for download is given as http://www.mooseframework.org/moose_packages/moose-environment-1_CentOSLinux-7.4.1708_x86_64.rpm. Unfortunately, the file is not available. Please provide the working URL for the download.

Thanks and regards,
Sandeep

**Comment by GiudGiud** [comment, marked as answer]
Hello

I dont think we use this method for deployment anymore. 
If this is for a computing cluster / HPC please follow these instructions
https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html
Otherwise those ones (easiest for smaller machines)
https://mooseframework.inl.gov/getting_started/installation/conda.html
or check this list for more options
https://mooseframework.inl.gov/getting_started/installation/

Guillaume

**Reply by sansugath** [reply]
Hi

Thank you for the information. I will look at your suggestions.
